pyEpiabm/pyEpiabm/sweep/host_progression_sweep.py

```diff
@@ -26,6 +25,20 @@ def _update_time_to_status_change(self):
         new_time = random.randint(1, 10)
         return new_time
 
+    def _set_latent_time(self):
+        """Calculates and returns latency period as calculated in
+        covid-sim to be given as time until next infection status
+        for a person who has been set as exposed.
+        """
```

Comment:
If this returns a value, then this needs to be documented - follow the `return/rtype` pattern in other files. Also maybe rename number to `_calc_latent_time` as you are calculating it but you don't set it (i.e. attribute it to the person) here?